libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -5594,7 +5594,8 @@ void GCS_MAVLINK::send_autopilot_state_for_gimbal_device() const
         0,      // velocity estimated delay in micros
         rate_ef_targets.z,  // feed forward angular velocity z
         est_status_flags,   // estimator status
-        0);     // landed_state (see MAV_LANDED_STATE)
+        0,      // landed_state (see MAV_LANDED_STATE)
+        std::numeric_limits<double>::quiet_NaN());  // [rad/s] Z component of angular velocity in NED (North, East, Down). NaN if unknown
```

Comment:
@peterbarker, are we happy get get NaN this way?  Normally I've created a const float higher up with the magic number.

Reply:
We could use AP::ahrs().get_yaw_rate_earth() to fill this in.

Reply:
Thanks, I fixed it like that!